src/rpc/gasEstimation.ts

```diff
@@ -488,251 +490,10 @@ export function simulateHandleOp(
 
     return simulateHandleOpV07(
         userOperation,
+        queuedUserOperations as UserOperationV07[],
         entryPoint,
         publicClient,
-        userOperation.sender,
-        userOperation.callData,
         entryPointSimulationsAddress,
         finalStateOverride
     )
 }
-
-function tooLow(error: string) {
```

Comment:
why are we removing this?

Reply:
These are not used anywhere, I don't think it's worth updating them

Reply:
ok!